SDL/SDL_gfx-2.0.25.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "SDL_gfx",
+    "//": "Code assumes that if __GNUC__ is defined and MMX is enabled, then x86-specific mmintrin.h is available",
+    "config-opts": [
+        "--disable-static",
+        "--disable-mmx"
+    ],
+    "build-options": {
+        "arch": {
+            "i386": {
+                "config-opts": [
+                    "--enable-mmx"
+                ]
+            },
```

Comment:
```suggestion
```

arch is no longer supported.

Reply:
I would actually remove the `disable-mmx` from config opts and just add it for `aarch64`. I feels less convoluted.

and I would submit a patch upstream to actuall properly detect architectures.

Reply:
I understand your point. FWIW I did it this way around because the `--enable-mmx` code will fail to compile on any non-x86 platform, so when someone comes along and adds RISC-V to Flathub, another arch-specific block would need to be added.

Reply:
when it happens, we'll see. Maybe by that time they'll have apply the patch I recommend submitting.

Reply:
I have to say I am not filled with enthusiasm at the prospect of sending a patch upstream. SDL_gfx (this one) appears to be abandoned; SDL2_gfx, its SDL2 successor, only marginally less so.